frontiers/location_in_the_neocortex_a_theory_of_sensorimotor_object_recognition_using_cortical_grid_cells/requirements.txt

```diff
@@ -0,0 +1 @@
+htmresearch==0.0.2
```

Comment:
Change it to version **0.0.3**

Reply:
the `pillow` package is required for plotting.
Add `pillow==5.2.0` to `requirements.txt`